app/javascript/components/admin/site_settings/appearance/BrandingImage.jsx

```diff
@@ -1,18 +1,51 @@
 import React from 'react';
-import { Card } from 'react-bootstrap';
-import { CloudUploadIcon } from '@heroicons/react/outline';
+import { Card, Stack } from 'react-bootstrap';
+import { CloudUploadIcon, TrashIcon } from '@heroicons/react/outline';
 import useUpdateSiteSetting from '../../../../hooks/mutations/admin/site_settings/useUpdateSiteSetting';
+import useSiteSettings from '../../../../hooks/queries/admin/site_settings/useSiteSettings';
 import FilesDragAndDrop from '../../../shared_components/utilities/FilesDragAndDrop';
+import Logo from '../../../shared_components/Logo';
+import Modal from '../../../shared_components/modals/Modal';
+import DeleteBrandingImageModal from './DeleteBrandingImageModal';
 
 export default function BrandingImage() {
   const updateSiteSetting = useUpdateSiteSetting('BrandingImage');
+  const { data: siteSettings } = useSiteSettings();
 
-  return (
-    <div className="mt-4">
-      <h6> Branding Image </h6>
+  if (siteSettings.BrandingImage.length > 1) {
+    return (
+      <div className="mb-3">
+        <h5> Branding Image </h5>
+        <Card className="border border-2 border-whitesmoke">
+          <Card.Body>
+            <div className="d-flex align-items-center">
+              <Stack>
+                <Logo />
+                <span> { siteSettings.BrandingImage } </span>
+              </Stack>
+              <div className="d-block ms-auto">
+                <Modal
+                  modalButton={<TrashIcon className="cursor-pointer hi-s" />}
+                  title="Are you sure?"
+                  body={<DeleteBrandingImageModal />}
+                />
+              </div>
+            </div>
+          </Card.Body>
+        </Card>
+      </div>
+    );
+  }
 
-      <FilesDragAndDrop numOfFiles={1} onDrop={(files) => updateSiteSetting.mutate(files[0])} formats={['.jpg', '.png']}>
-        <Card className="border-0 shadow-sm mt-3 text-center">
+  return (
+    <div className="mb-3">
```

Comment:
This also may be extracted into a component?

Reply:
Yeah I thought about making this a component since we are using something similar in Room/Presentation.
But this will have to be for another PR.

Reply:
Of course, that's what I meant